core/types/transactions/payloads.go

```diff
@@ -403,11 +270,16 @@ func (v *ValidatorVoteIDs) UnmarshalBinary(p0 serialize.SerializedData) error {
 // ValidatorVoteBodies is a payload for submitting the full vote bodies for any resolution.
 type ValidatorVoteBodies struct {
 	// Events is an array of the full resolution bodies the caller is voting for.
-	Events []*types.VotableEvent
+	Events []*VotableEvent
 }
 
 var _ Payload = (*ValidatorVoteBodies)(nil)
 
+type VotableEvent struct {
```

Comment:
Just moved I know, but docs would be good here.  Not obvious what are types and the meaning of body (saved in event store and interpreted by some registered oracle, etc.).